pkgs/development/libraries/pipewire/default.nix

```diff
@@ -107,10 +106,10 @@ let
       # Place SPA data files in lib output to avoid dependency cycles
       ./0095-spa-data-dir.patch
 
-      # remove when updating to 0.3.60
-      (fetchpatch { # filter-chain: iterate the port correctly
-        url = "https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/94a64268613adac8ef6f3e6c1f04468220540d00.patch";
-        sha256 = "sha256-IDTB7NgadgR3vKv97Nvd9pBfnOnMi21YsvLdD1Ew7HE=";
+      # FIXME: remove in 0.3.61
+      (fetchpatch {
```

Comment:
Would be nice to add the message of the commit being backported or at least a short explanation why the patch is needed. (the patch itself is good, I was going to suggest pulling it in before I saw you'd already done that)

Reply:
It was recommended on Matrix and also on the DISTROS entry in the changelog.

Reply:
Yeah I just mean it's nice to have the reason or the commit msg repeated here so anyone can see at a glance what it is about without consulting release notes or fetching & reading the patch itself.

Reply:
Just pushed it.